### PR TITLE
feat(vocabularies): Activate DELETE for vocabularies endpoint

### PR DESCRIPTION
**Pull request by ioanpocol**

### CODE REVIEW

File: superdesk/vocabularies/vocabularies.py [hunk 1]
```diff
@@ -73,7 +73,6 @@ class VocabulariesResource(Resource):
         },
         'service': {
             'type': 'dict',
-            'default': {'all': True},
```
Comment (petrjasek) [inline superdesk/vocabularies/vocabularies.py]: why is this removed? @ioanpocol

Comment (ioanpocol) [inline superdesk/vocabularies/vocabularies.py]: Duplicates genre and other dropdowns that had default vocabulary. If the service is present means that it is a custom vocabulary.

Comment (petrjasek) [inline superdesk/vocabularies/vocabularies.py]: ok, this will need some ui changes so that new vocabularies will show up in profile management
